test/IntelFPGAMemoryAttributes.ll

```diff
@@ -13,19 +13,25 @@
 ; CHECK-SPIRV: Decorate {{[0-9]+}} MaxconcurrencyINTEL 4
 ; CHECK-SPIRV: Decorate {{[0-9]+}} SinglepumpINTEL
 ; CHECK-SPIRV: Decorate {{[0-9]+}} DoublepumpINTEL
+; CHECK-SPIRV: Decorate {{[0-9]+}} MaxReplicatesINTEL 2
+; CHECK-SPIRV: Decorate {{[0-9]+}} SimpleDualPortINTEL
+; CHECK-SPIRV: Decorate {{[0-9]+}} MergeINTEL "foo" "depth"
```

Comment:
Please ensure that corresponding `OpCapability` and `OpExtensions` instructions are emitted

Reply:
Added check for FPGAMemoryAttributesINTEL capability.
Extension instruction is not emitted now, will fix in other commit.